lib/api/products.js

```diff
@@ -81,11 +81,12 @@ export function applyProductRevision(product) {
  */
 export function variantIsSelected(variantId) {
   const current = ReactionProduct.selectedVariant();
-  if (current && typeof current === "object" && (variantId === current._id || ~current.ancestors.indexOf(variantId))) {
-    return true;
-  }
-
-  return false;
+  if (current.ancestors) {
```

Comment:
You've introduced a possible failure here because you are checking for `current.ancestors` before checking if `current` exists as is done below.